src/tools/create-lambda-handler.js

```diff
@@ -19,11 +19,39 @@ const environmentTools = require('./environment');
 const schemaTools = require('./schema');
 const ZapierPromise = require('./promise');
 
+const RequestSchema = require('zapier-platform-schema/lib/schemas/RequestSchema');
+const FunctionSchema = require('zapier-platform-schema/lib/schemas/FunctionSchema');
+
+const isRequestOrFunction = obj => {
+  return (
+    RequestSchema.validate(obj).valid || FunctionSchema.validate(obj).valid
+  );
+};
+
 const extendAppRaw = (base, extension) => {
-  const concatArray = (objValue, srcValue) => {
+  const keysToOverride = [
+    'test',
+    'perform',
+    'performList',
+    'performSubscribe',
+    'performUnsubscribe'
+  ];
+  const concatArray = (objValue, srcValue, key) => {
```

Comment:
We should probably rename `contactArray` to something else as the function doesn't only do array concatenation anymore.